_pages/en_US/homebrew-launcher-(ninjhax2-dx).txt

```diff
@@ -0,0 +1,55 @@
+---
+title: "Homebrew Launcher (ninjhax2-dx)"
+---
+
+{% include toc title="Table of Contents" %}
+
+{% capture technical_info %}
+<summary><em>Technical Details (optional)</em></summary>
+[Ninjhax2](https://github.com/smealum/ninjhax2.x) is an exploit for the game "Cubic Ninja" made by [smealum](https://github.com/smealum). This page uses an updated version of ninjhax by [zoogie](https://github.com/zoogie) called [ninjhax2-dx](https://github.com/zoogie/ninjhax2-dx), which supports 11.17.0.
+{% endcapture %}
+<details>{{ technical_info | markdownify }}</details>
+{: .notice--info}
+
+This method requires a working 3DS camera.
+{: .notice--info}
+
+This process will overwrite your Cubic Ninja save file, if you have one.
+{: .notice--warning}
+
+## What You Need
+* The game "Cubic Ninja"
+* The latest release of [ninjhax2-dx](https://github.com/zoogie/ninjhax2-dx/releases)
+* The latest release of [Luma3DS](https://github.com/LumaTeam/Luma3DS/releases/latest) (the Luma3DS `.zip` file)
+
+## Instructions
+
+### Section I - Prep Work
+
+1. Power off your device
+1. Insert your SD card into your computer
+1. Copy `Launcher.dat` from the folder appropriate for your console from the ninjhax2-dx `.zip` file to the root of your SD card
+  + The root of the SD card refers to the initial directory on your SD card where you can see the Nintendo 3DS folder, but are not inside of it
+1. Copy `boot.firm` and `boot.3dsx` from the Luma3DS `.zip` to the root of your SD card
+
+### Section II - ninjhax2-dx
+1. Reinsert your SD card into your device
+1. Power on your device
+1. Launch "Cubic Ninja" once and select "Create"
+  + If prompted, delete all data
+1. Select "QR Code"
+1. Select "Scan QR Code"
+1. On your PC, open the browser and go to [ninjhax2-dx website](https://zoogie.github.io/web/nh2dx/)
```

Comment:
super minor thing but add a the, "go to the ninjhax2-dx website" sounds more natural to me imo